.github/workflows/ci.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
I suggest to use a proper name for the workflow i.e. `hadolint` and a filename like `hadolint.yml`